### ISSUE TITLE
The word "submission" is ambiguous

### ISSUE THREAD

**Issue by rghsh** [ProFormA/proformaxml#4]
The word "submission" is used to name a student's solution file and to name the zip or xml file handed from the LMS to the grader. The latter "submission" file contains the former "submission" file. We should find a better name for the latter concept. Proposals: request, job, order, ... The new name should e used consistently in the white paper and in the xsd.

**Comment by upriss** [comment]
Does this affect the xsd at all? Maybe we can still change the whitepaper? Maybe we should make a separate list of additional requirements that are not in the xsd. I suspect most people won't read the whole whitepaper.

**Comment by rghsh** [comment]
Yes, this would affect the xsd. We would replace the xsd type "submission-type" with something like "request-type". The root element would be &lt;request&gt; instead of &lt;submission&gt;.

**Comment by upriss** [comment]
But that would not be backward compatible. It would be much easier to change the other uses of the word "submission". We have used the terms "file(s) submitted by student", "student submitted code" and so on in many places in the whitepaper. We could check whether that is done consistently.
